From f1f9bcbcfc227cec282d28fc4979120e1cd86170 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jan=20Jan=C3=9Fen?= <janssen@mpie.de>
Date: Sun, 9 Jul 2023 12:39:23 -0600
Subject: [PATCH] fix setup

---
 setup.py | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/setup.py b/setup.py
index cfcff81..b50c749 100644
--- a/setup.py
+++ b/setup.py
@@ -17,6 +17,13 @@
     author_email="jan.janssen@outlook.com",
     license="BSD",
     packages=find_packages(exclude=["*tests*"]),
+    package_data={
+        'webapp/static/css': ['*.css'],
+        'webapp/static/fonts/poppins': ['*.ttf'],
+        'webapp/static/images': ['*.jpg'],
+        'webapp/static/images/icons': ['*.ico'],
+        'webapp/templates': ['*.html'],
+    },
     install_requires=[
         "google-api-python-client==2.92.0",
         "google-auth==2.21.0",
